systemd/systemd.go

```diff
@@ -437,12 +437,12 @@ func MountUnitPath(baseDir string) string {
 }
 
 func (s *systemd) WriteMountUnitFile(name, what, where, fstype string) (string, error) {
-	extra := ""
+	options := []string{"nodev"}
```

Comment:
on classic we mount /dev from the real root on top of the core snap anyway, no?
on core the mounting of core on / is not done by this unit I think?

so do we really need to have real devices under /snap/core/current/dev ?